data/human/deep missions.txt

```diff
@@ -4249,3 +4249,15 @@ mission "The Book of Skadenga"
 			label ignore
 			`	You keep walking. You already know that story.`
 				decline
+
+# resets any negative "deep" reputation in older saves
+mission "Deep Government Reputation Reset"
+	invisible
+	to offer
+		"reputation: Deep" <= -1000
+		or
+			not "chosen sides"
+			has "main plot completed"
```

Comment:
If we sync with the Republic's reputation, then we don't really need any offer conditions.
```suggestion
```